### ISSUE TITLE
C:/Ruby24/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- bundler/setup (LoadError)

### ISSUE THREAD

**Issue by imvieira** [travis-ci/travis.rb#569]
```
p\travis.rb\bin>ruby travis
C:/Ruby24/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- bundler/setup (LoadError)
        from C:/Ruby24/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from travis:6:in `<main>'
```

**Comment by 0x69676f72** [comment]
`C:\Users\IgorMonteiroVieira\Desktop\travis.rb\bin>ruby travis
C:/Ruby24/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- bundler/setup (LoadError)
        from C:/Ruby24/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from travis:6:in `<main>'`

**Comment by 0x69676f72** [comment]
```
C:\Users\IgorMonteiroVieira\Desktop\travis.rb\bin>ruby travis
C:/Ruby24/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- bundler/setup (LoadError)
        from C:/Ruby24/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from travis:6:in `<main>'
```

**Comment by 0x69676f72** [comment]
```
\travis.rb\bin>ruby -v
ruby 2.4.2p198 (2017-09-14 revision 59899) [i386-mingw32]
```

**Comment by 0x69676f72** [comment]
How to start travis on windows?

**Comment by imvieira** [comment]
> <@W7BS4DL6P> Hi,   If apply Apply Fix Pack 9.0.0.9   i think cover all this fix , correct ?  <@W3PHMUFJL> <@W3QVAGSCX>
<sub>[View in Slack](https://ibm-cio.slack.com/archives/C9E8Y9QSD/p1539189793000100)</sub>

**Comment by imvieira** [comment]
> <@W7BS4DL6P> Hi,   If apply Apply Fix Pack 9.0.0.9   i think cover all this fix , correct ?  <@W3PHMUFJL> <@W3QVAGSCX>
<sub>[View in Slack](https://ibm-cio.slack.com/archives/C9E8Y9QSD/p1539189793000100)</sub>
